ee/agent/storage/bbolt/stores_bbolt.go

```diff
@@ -33,6 +33,7 @@ func MakeStores(ctx context.Context, slogger *slog.Logger, db *bbolt.DB) (map[st
 		storage.ServerProvidedDataStore,
 		storage.TokenStore,
 		storage.ControlServerActionsStore,
+		storage.LauncherHistoryStore,
```

Comment:
Wow. Looking at the stores here, I'm not sure there was an obvious one. But creating a new one to hold a single value feels big. Done now though